projects/extension/sql/idempotent/015-voyageai.sql

```diff
@@ -0,0 +1,48 @@
+-------------------------------------------------------------------------------
+-- voyageai_embed
+-- generate an embedding from a text value
+-- https://docs.voyageai.com/reference/embeddings-api
+create or replace function ai.voyageai_embed
+( model text
+, input_text text
+, input_type text default null
```

Comment:
It doesn't look like you're using the `input_type` param anywhere. And why not default to "document"?

Reply:
Oof, this was a mistake on my part. I reworked this one too many times and left it incomplete. Fixed.